docs/source/reference/migration/from-router-v1.mdx

```diff
@@ -189,6 +189,22 @@ limits:
 
 - `telemetry.instrumentation.spans.mode` becomes `spec_compliant` by default instead of `deprecated`.
 
+### Renamed Metrics
+
+Some of the older metrics in the router were not using the most recent `.` naming convention and were instead separated by `_`. For consistency purposes, the following metrics are renamed to use the up-to-date `.` notation:
+
+- `apollo_router_opened_subscriptions` -> `apollo.router.opened.subscriptions`
+- `apollo_router_cache_hit_time` -> `apollo.router.cache.hit.time`
+- `apollo_router_cache_size` -> `apollo.router.cache.size`
+- `apollo_router_cache_hit_count` -> `apollo.router.cache.hit.count`
+- `apollo_router_cache_miss_time` -> `apollo.router.cache.miss.time`
+- `apollo_router_cache_miss_count` -> `apollo.router.cache.miss.count`
+- `apollo_router_state_change_total` -> `apollo.router.state.change.total`
+- `apollo_router_span_lru_size` -> `apollo.router.exporter.span.lru.size`
+- `apollo_router_session_count_active` -> `apollo.router.session.count.active`
+- `apollo_router_uplink_fetch_count_total` -> `apollo.router.uplink.fetch.count.total`
+- `apollo_router_uplink_fetch_duration_seconds` -> `apollo.router.uplink.fetch.duration.seconds`
```

Comment:
nit: maybe use a markdown table here?

Reply:
oh yea, good idea!

Reply:
done in 4b931a7b48d7ebaa3ff94a6913ef96c6d8381133